src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -52,55 +52,47 @@ private BoundExpression BindIsPatternExpression(IsPatternExpressionSyntax node,
             // Note that these labels are for the convenience of the compilation of patterns, and are not necessarily emitted into the lowered code.
             LabelSymbol whenTrueLabel = new GeneratedLabelSymbol("isPatternSuccess");
             LabelSymbol whenFalseLabel = new GeneratedLabelSymbol("isPatternFailure");
-
-            bool negated = pattern.IsNegated(out var innerPattern);
             BoundDecisionDag decisionDag = DecisionDagBuilder.CreateDecisionDagForIsPattern(
```

Comment:
This file is simply reverted to the state before https://github.com/dotnet/roslyn/pull/49369